.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Format to report a bug
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
```

Comment:
Sorry I didn't notice before, the link would be useful here too

Reply:
Done